index.bs

```diff
@@ -552,18 +552,43 @@ The header's ABNF is:
 The 'Sec-CH-UA-Form-Factor' Header Field {#sec-ch-ua-form-factor}
 ------------------------------
 
-The <dfn http-header>`Sec-CH-UA-Form-Factor`</dfn> request header field gives a server information
-about the [=user agent=]'s [=form-factor=]. It is a [=Structured Header=] whose value MUST be a
-[=structured header/string=]. Its value SHOULD match one of the following common form-factor values:
-"Automotive", "Mobile", "Tablet", "TV", "VR", "XR", "Unknown" or the empty string.
-[[!RFC8941]].
+The <dfn http-header>`Sec-CH-UA-Form-Factor`</dfn> request header field gives a
+server information about the [=user agent=]'s [=form-factor=]. It is a
+[=Structured Header=] whose value MUST be a [=structured header/list=]
+[[!RFC8941]]. The header's values MUST be given in lexical order.
```

Comment:
This feels like a `Note:` more than a standalone paragraph

Reply:
I think `Note:` is non-normative, but I've used MUST here. Are you suggesting  making it non-normative?

Reply:
Good point, nvm then